README.md

```diff
@@ -82,8 +82,6 @@ Here is the table for the support timeline:
 
 ### ansible-core
 
-- stable-2.12
-- stable-2.13
```

Comment:
I would do all not release related changes in a separate PR so we can backport them later. Not a big deal but it's a good practice to follow

Reply:
@Andersson007 my bad, I didn't sync my fork after the merge of the PR that remove stable-2.12 and 13. It's now fixed.